README.md

```diff
@@ -1,2 +1,281 @@
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Visual Studio Code.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Making sure the repo is ready
+
+- **[Make sure that there are no linter errors]**
+- **[Make sure that you used correct (GitHub flow)]**
+- **[Make sure that you documented your work]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Cooming Soon.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Set up a "Hello Microverse" project
+Learning Objectives
+Create branches with different base branches.
+Understand the concept of version control.
+Follow GitHub flow.
+Open pull requests in GitHub.
+Write descriptive, easy to understand commit messages.
+Write short, detailed, easy to understand descriptions of pull requests.
+Write a descriptive and easy to understand README in business English for every project submitted.
+Demonstrate an ability to submit a project for code review..
+
```

Comment:
- [ ] As mentioned by the previous reviewer, you should add meaningful information for intending users of this repository. The text you place here is not easy to follow. Remove this text and leave that place blank because it serves as a header for the prerequisite, 

![image](https://user-images.githubusercontent.com/84156714/226621855-f51465e9-d832-4e5d-8ec7-125a96250681.png)

- For the Prerequisite Section, kindly add tools that are needed before you can clone this repository on a local machine. Tools like:

    - Installation of Git
    - Installation of NPM
    - A working browser
    - Internet connection
    - A code editor like Visual Studio Code
    - Installation of Live Server plugging in Visual Studio Code
    - Any other tool you can think about

- For the Setup section, kindly add instructions for cloning your repository on a local machine 👍
- For the Install section, kindly add instructions for setting up linters used in this project
- For the Usage section, include instructions for opening this project on a browser. For example, opening the project by launching the Live Server plugging in visual studio code or any other means that you know.
- For Test, it should be removed as it is not necessary for this project
- For Deployment, it should be removed as it is also not necessary for this project.

See [this repository](https://github.com/KenbonTN/Hello-Microverse-1/tree/feature-branch) as a guide on editing your README. **_Note that this student did not edit the future section properly. ensure you add at least three future features. Give special attention to the getting started section_**

Reply:
I wasn't even paying attention to the look of the readme file. I didn't even notice the importance of it, being something like the face of the project, I see it now... 
I've added/modified everything asked, except some of the Instructions. These were absorbed by the future features section (For now). I see that I only have one request review left, so it would be great if you can give me some feedback about the evolution of my project. Thank you very much!